raiden/network/transport/matrix/transport.py

```diff
@@ -1134,19 +1170,9 @@ def _handle_text(self, room: Room, message: MatrixMessage) -> List[Message]:
 
         return validate_and_parse_message(message["content"]["body"], peer_address)
 
-    def _handle_sync_messages(self, sync_messages: MatrixSyncMessages) -> bool:
-        """ Handle text messages sent to listening rooms """
-        if self._stop_event.ready():
-            return False
-
-        assert self._raiden_service is not None, "_raiden_service not set"
-
-        all_messages: List[Message] = list()
-        for room, room_messages in sync_messages:
-            # TODO: Don't fetch messages from the broadcast rooms. #5535
-            if not self._is_broadcast_room(room):
-                for text in room_messages:
-                    all_messages.extend(self._handle_text(room, text))
+    def _process_messages(self, all_messages: List[Message]) -> None:
+        log.debug("raiden_service", raiden_service=self._raiden_service)
```

Comment:
Could be removed.

Reply:
What do you mean by that?

Reply:
I felt this log line doesn't give much information